### ISSUE TITLE
Code Cleanup - `i` tag used for icons bad practice

### ISSUE THREAD

**Issue by mheppler** [IQSS/dataverse#6144]
Accessibility reports from Siteimprove warned that using `i` italics-tags for icons is considered "bad practice".

```
<i class="icon-dataverse text-icon-inline"></i>
```

<img width="238" alt="Screen Shot 2019-09-06 at 9 03 34 AM" src="https://user-images.githubusercontent.com/687227/64429975-3f230c00-d085-11e9-8e7e-cdba80f8181e.png">

These tags were randomly used with the custom icons created for dataverses, datasets and files on the following pages:

- dataverse_header (navbar)
- dataverse (edit btn dropdown) + search_include (type facets)
- dataverseuser (notifications)
- maintanence (navbar)
- mydata (cards)

They should be replaced with `span` tags, as the Bootstrap icons utilize.

**Comment by mheppler** [comment]
Also included in this PR was the addition of an alt attribure for the custom logo image in the navbar, and "home page" in the bundle. Minor expansion of scope, as I was in the dataverse_header file code. A larger effort to address alt attributes for images across the application will be made in #6106.
